bin/tectonics-modeling-L0/session1-aspect.ipynb

```diff
@@ -0,0 +1,2840 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    import io 
</pre><p>Are you using this library anywhere? If not, then you can remove this.</p><p>Also true for the three subsequent lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>